src/testRunner/unittests/publicApi.ts

```diff
@@ -19,21 +18,6 @@ describe("unittests:: Public APIs", () => {
         it("should be acknowledged when they change", () => {
             Harness.Baseline.runBaseline(api, fileContent, { PrintDiff: true });
         });
-
-        it("should compile", () => {

```

Comment:
We actually already test this by nature of the new public API compiler tests I added, which disable skipLibCheck to test the actual public API. This change could be pulled out into another PR but it's needed for this PR because this test assumes that our d.ts files are self-contained, which is no longer true.

Alternatively, I can just make `tsserverlibrary.d.ts` a copy of `typescript.d.ts` and it'd be fine.

Reply:
is is possible to create symlinks for tsserverlibrary.js and tsserverLibrary,d,ts to point to typescript.js and typescript.d.ts instead

Reply:
Right now `tsserverlibrary` just reexports `typescript`, it's just that it's not self contained so this one particular test breaks (but isn't needed anyway).

In terms of the package itself, we can't really use symlinks as those are not universally supported on Windows or by package managers themselves. There are ways that people store packages which can't contain symlinks at all (e.g., zips, used in yarn, for example).